{"uuid":"d9e6e68a-75a3-4261-97bd-8c4c8e248ea5","historyId":"32ddd832eec4c61076355fdb3b07fbcf","fullName":"demoCRM_Web.testScripts.TC001_Create_NewContact.create_NewCustomer","labels":[{"name":"package","value":"demoCRM_Web.testScripts.TC001_Create_NewContact"},{"name":"testClass","value":"demoCRM_Web.testScripts.TC001_Create_NewContact"},{"name":"testMethod","value":"create_NewCustomer"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"demoCRM_Web.testScripts.TC001_Create_NewContact"},{"name":"host","value":"YCSLAP1132"},{"name":"thread","value":"23576@YCSLAP1132.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"critical"}],"links":[],"name":"create_NewCustomer","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.RuntimeException: Error reading Excel data: Sheet not found: create_NewCustomer","trace":"java.lang.RuntimeException: java.lang.RuntimeException: Error reading Excel data: Sheet not found: create_NewCustomer\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:809)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:734)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\nCaused by: java.lang.RuntimeException: Error reading Excel data: Sheet not found: create_NewCustomer\r\n\tat cogmentoCRM_Web.dataUtils.ExcelUtility.getExcelDataAsMap(ExcelUtility.java:68)\r\n\tat cogmentoCRM_Web.dataUtils.ExcelUtility.dynamicExcelData(ExcelUtility.java:25)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)\r\n\t... 25 more\r\n"},"stage":"finished","description":"Creates a New Contact","steps":[],"attachments":[],"parameters":[],"start":1747824840766,"stop":1747824840768}